### DISCUSSION TITLE
✅ 🔗 Text and Icon are not aligned in the Button widget (Duplicate of #16511)

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21236] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

The button widget is a very useful widget. But unfortunately, it has very elementary defects. One of the most important is that we cannot set the icon size in the button widget. This is really surprising! But in this issue, I don't want to talk about the icon size feature in the button widget. I have a very simple request. very simple.

If we increase the size of the icon for any reason, the text will not be aligned with the icon.

![b](https://user-images.githubusercontent.com/103376243/216786572-f552578d-622d-4377-924c-0dd1ac63892c.png)

### Describe the solution you'd like

In the button widget, to the **elementor-button-content-wrapper** class, give the **align-items: center;** attribute. This ensures that the text and icon in the button widget are aligned under any circumstances.

Team Elementor, it's really great that you are working on big features now. But please don't forget the basic features and requirements.

**Comment by breakerh** [comment]
I’d like to highlight this feature request. Besides alignment issues, it also lacks separate color and size options. Currently, this requires custom CSS unnecessarily, even though the style setup exists for other elements that may or may not include icons. Duplicating the CTA icon styling with a conditional to hide it if no icon is selected would be quite easy to do, right?

**Comment by nicholaszein** [comment]
Hello there @Joliamendi!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.28.0! 🥳 

Now all button icons across different Elementor elements are correctly aligned vertically! 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
